app/forms/nsm/steps/work_item_form.rb

```diff
@@ -27,7 +27,7 @@ def allow_uplift?
 
       def apply_uplift
         allow_uplift? &&
-          (@apply_uplift.nil? ? uplift.present? : @apply_uplift == 'true')
+          (@apply_uplift.nil? ? (uplift.present? && uplift.positive?) : @apply_uplift == 'true')
```

Comment:
Could shorten a bit:
```suggestion
          (@apply_uplift.nil? ? (uplift.to_f.positive?) : @apply_uplift == 'true')
```

nil.to_f.positive? => false
0.0001.to_f.positive? => true